### DISCUSSION TITLE
bzip2: Cannot exec: No such file error in micromamba install

### DISCUSSION THREAD

**Discussion by KMasch** [biostars/biostar-handbook-issues#285] [Q&A]
Hello,

While trying to install the Biostar Handbook on a windows computer, I got the following text:

# 2/10 Installing the micromamba manager
#
tar (grandchild): bzip2: Cannot exec: No such file or directory
tar (grandchild): Error is not recoverable: exiting now
tar: Child died with signal 13
tar: Error is not recoverable: exiting now

I didn't see this error in the handbook so I tried installing only micromamba and got the same issue. Is there something I missed from a previous step?

**Comment by ialbert** [comment]
follow the linux install, looks like bzip2 is not installed on your system, it is mentioned in the Linux setup section of the latest version of the online book

**Reply by KMasch** [reply]
Thank you for your help, that fixed my issue.
